about.css

```diff
@@ -0,0 +1,579 @@
+@media screen and (max-width: 767px) {
+  * {
+    margin: 0;
+    padding: 0;
+    font-family: 'COCOGOOSE', sans-serif;
+  }
+
+  main {
+    margin: 0;
+    padding: 0;
+    background-image: url('IMAGES/main_background.png');
+    background-repeat: no-repeat;
```

Comment:
- [ ] Please move the content more to the bottom following the mobile version for about page [design guideline](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) but first remove the navbar from the main tag as I suggested and then arrange items. Keep shining 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163674840-d953438e-5a56-430a-95b0-80241179fc88.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163674860-9481bbd2-f003-4de7-b505-f29b87090c5c.png)